gradle-baseline-java-config/resources/idea/intellij-java-palantir-style.xml

```diff
@@ -44,6 +44,9 @@
             </value>
           </option>
         </GroovyCodeStyleSettings>
+        <JavaCodeStyleSettings>
```

Comment:
will this affect what IntelliJ and Eclipse do? I don't think it's cool for autoformat to cause failures